erpnext/subcontracting/doctype/subcontracting_receipt/test_subcontracting_receipt.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2022, Frappe Technologies Pvt. Ltd. and Contributors
+# See license.txt
+
+# import frappe
+import unittest
+
+
+class TestSubcontractingReceipt(unittest.TestCase):
```

Comment:
no tests at all for such massive change?

Reply:
> no tests at all in full PR for such massive change?

Will add tests for all at once.

Reply:
That step usually gets skipped when deadlines are near. Always do it progressively as changes are introduced! 😄